cmd/conduit/root/root.go

```diff
@@ -50,8 +50,9 @@ type RootFlags struct {
 }
 
 type RootCommand struct {
-	flags  RootFlags
-	output ecdysis.Output
+	DefaultCfg conduit.Config
```

Comment:
I personally think this would look better if the field were named simply `Cfg`. This way, it doesn't necessarily imply that the configuration used here is the default (it'll most likely be), but technically speaking, it could be any other configuration.

```suggestion
	Cfg conduit.Config
```

Reply:
I like the suggestion, I'll commit it.:)